src/studio/src/designer/backend/Infrastructure/GitRepository/GitRepository.cs

```diff
@@ -104,7 +105,17 @@ public async Task<string> ReadTextByRelativePathAsync(string relativeFilePath)
             // and that the BOM bytes isn't removed on read in the ReadTextAsync method.
             // Should try to fix this as this method is more performant than ReadAllTextAsync.
             // return await ReadTextAsync(absoluteFilePath)
-            return await File.ReadAllTextAsync(absoluteFilePath, Encoding.UTF8);
+            File.SetAttributes(absoluteFilePath, FileAttributes.Normal);
+
+            try
+            {
+                return await File.ReadAllTextAsync(absoluteFilePath, Encoding.UTF8);
+            }
+            catch (IOException)
+            {
+                Thread.Sleep(1000);
```

Comment:
Did this help?

Reply:
yes, based on the code coverage analysis the clause is actually used when running the unit tests on the server, although I cannot see the same behavior when debugging the tests locally

Reply:
but then the next run after making a test method static causes the tests to fail again, so I don't really know..